src/utils/DragAndDrop.js

```diff
@@ -174,6 +186,18 @@ define(function (require, exports, module) {
             event = event.originalEvent || event;
 
             var files = event.dataTransfer.files;
+
+            var types = event.dataTransfer.types;
```

Comment:
Might be worth it to move these blocks into a function call such as `stopURIListPropagation()` or similar instead of repeating the blocks of code.